app/models/provisioning_template.rb

```diff
@@ -124,13 +124,13 @@ def self.global_template_name_for(kind, renderer)
   end
 
   def self.build_pxe_default(renderer)
-    return [422, _("No TFTP proxies defined, can't continue")] if (proxies = SmartProxy.with_features("TFTP")).empty?
+    return [:unprocessable_entity, _("No TFTP proxies defined, can't continue")] if (proxies = SmartProxy.with_features("TFTP")).empty?
```

Comment:
changed all numeric return codes to rails symbols while i'm touching this function.